tls/s2n_connection.c

```diff
@@ -633,31 +633,33 @@ int s2n_connection_wipe(struct s2n_connection *conn)
 int s2n_connection_set_recv_ctx(struct s2n_connection *conn, void *ctx)
 {
     POSIX_ENSURE_REF(conn);
-    POSIX_GUARD(s2n_connection_free_recv_io_context(conn));
+    POSIX_GUARD(s2n_connection_free_managed_recv_io(conn));
     conn->recv_io_context = ctx;
-    return 0;
+    return S2N_SUCCESS;
 }
 
 int s2n_connection_set_send_ctx(struct s2n_connection *conn, void *ctx)
 {
     POSIX_ENSURE_REF(conn);
-    POSIX_GUARD(s2n_connection_free_send_io_context(conn));
+    POSIX_GUARD(s2n_connection_free_managed_send_io(conn));
     conn->send_io_context = ctx;
     return 0;
```

Comment:
This is the only one you didn't update ;)

```suggestion
    conn->send_io_context = ctx;
    return S2N_SUCCESS;
```

Reply:
🤭